added configpath parameter and delete method

mrvnklm opened the pull request:
- added **setConfigPath(CONFIGPATH)** to change the destination and filename for the generated config file
- added **delete()** method to delete config file afterwards (for temporary usage)

Commit:
added configpath parameter and delete method

- added **setConfigPath(CONFIGPATH)** to change the destination and filename for the generated config file
- added **delete()** method to delete config file afterwards (for temporary usage)

mrvnklm commented:
@DivineOmega Please merge if you're fine with my changes. (Sorry for previous PRs)

Jord-JD commented on src/ConfigGenerator.php:
```diff
@@ -15,6 +15,7 @@ class ConfigGenerator
         'logpath' => '\'.__DIR__.\'/log.txt',
         'logfunction' => '',
         'muteLog' => 'false',
+        'configpath' => null
```
```suggestion
         'configpath' => __DIR__.'/../configs/'
```

Could we default this value as suggested so the ternary ifs are not required?

mrvnklm replied:
That's the way I made the first PR, but then I realized, that I need completely random filenames from time to time. We could use a **setFilename** or **setRandomFilename** method or what would be your approach?

Jord-JD replied:
To avoid breaking changes, I think we should keep the config path defaulted as suggested.

You could also have a `setFilename` method that sets a private member variable, which if set, could be used as an alternate to the `generateFilename()` method.

mrvnklm replied:
I updated the code, hope you are fine with the changes I've made

Jord-JD replied:
This looks good.

However, I've just noticed the `delete` method does not actual appear to delete the file?

mrvnklm replied:
True, sorry for that. I fixed it.

Jord-JD replied:
Excellent. I will get this merged and release a new version.

Jord-JD reviewed (changes requested):
Thanks for the contribution. I've made a quick suggestion. Curious what you think.

Commit:
updated with setFilename method

deleted exception for delete() method - no need to throw an exception here, if file was already deleted

Commit:
fixed delete method to actually delete file

Jord-JD commented:
Released as part of version 1.2.0. Thank you. :+1:
